pkg/controller/pravega/pravega_segmentstore.go

```diff
@@ -324,6 +326,25 @@ func configureSegmentstoreTLSSecret(podSpec *corev1.PodSpec, p *api.PravegaClust
 	}
 }
 
+func configureECSTLSSecret(podSpec *corev1.PodSpec, p *api.PravegaCluster) {
+	if p.Spec.TLS.IsSecureECS() {
+		vol := corev1.Volume{
+			Name: tlsVolumeName,
+			VolumeSource: corev1.VolumeSource{
+				Secret: &corev1.SecretVolumeSource{
+					SecretName: p.Spec.TLS.Static.EcsSecret,
+				},
+			},
+		}
+		podSpec.Volumes = append(podSpec.Volumes, vol)
+
+		podSpec.Containers[0].VolumeMounts = append(podSpec.Containers[0].VolumeMounts, corev1.VolumeMount{
+			Name:      tlsVolumeName,
+			MountPath: tlsMountDir,
+		})
```

Comment:
Has this change been tested with TLS for SegmentStore enabled and TLS for ECS also enabled?
I see that both the secrets use the same mount path - "tlsMountDir". This won't work when SegmentStore TLS is also enabled, since 2 secrets can't be mounted to the same path in the same container, unless you use a projected volume : https://kubernetes.io/docs/tasks/configure-pod-container/configure-projected-volume-storage/